ansible_collections/arista/avd/molecule/eos_validate_state/intended/test_catalogs/dc1-leaf1b-catalog.yml

```diff
@@ -1,7 +1,7 @@
 anta.tests.connectivity:
 - VerifyLLDPNeighbors:
     neighbors:
-    - neighbor_device: dc1-leaf1a
+    - neighbor_device: dc1-leaf1a.dc1.local
```

Comment:
so given that this leaf is not deployed :) should we REALLY be running any tests ? ;) - this should probably be a new PR to tackle this very simple use case

Reply:
doh - https://github.com/aristanetworks/ansible-avd/pull/3272

Reply:
:)